apis/scheduling/config/types.go

```diff
@@ -123,3 +123,14 @@ type ReservationArgs struct {
 	// EnablePreemption indicates whether to enable preemption for reservations.
 	EnablePreemption *bool `json:"enablePreemption,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// GangArgs defines the parameters for Gang Scheduling plugin.
+type GangArgs struct {
```

Comment:
The new plugin args must be register in register.go include `apis/scheduling/config` and `apis/scheduling/config/v1beta2`

Reply:
done